webapp/src/ui/page.js

```diff
@@ -1,56 +1,39 @@
-import React from 'react'
-import {connect} from 'react-redux'
-import {
-  Card,
-  CardContent,
-  Grid,
-  Paper,
-} from '@material-ui/core'
+import React from 'react';
+import {connect} from 'react-redux';
+import {Grid, Typography, Link} from '@material-ui/core';
 
 import Courses from './courses';
 
+const Page = () => (
+	<Grid container spacing={2}>
+		<Grid item xs={12}>
+			<Typography gutterBottom variant="h2">
+				Approved Transfer Courses
+			</Typography>
 
-const cardStyle = {
-  fontSize: 40,
-  justifyContent: 'center',
-  alignContent: 'center',
-  textAlign: 'center'
-};
+			<Typography gutterBottom variant="h4">
+				at <Link href="https://mtu.edu" target="_blank" rel="noopener noreferrer">Michigan Tech</Link>
+			</Typography>
 
-class Page extends React.Component {
-  render() {
-    return (
-      <Grid container spacing={2}>
-        <Grid item xs={12}>
-          <Paper>
-            <Card>
-              <CardContent style={cardStyle}>
-                Course Gather
-              </CardContent>
-            </Card>
-          </Paper>
-        </Grid>
-        <Grid item xs={12}>
-          <Paper>
-            <Card>
-              <CardContent style={cardStyle}>
-                <Courses />
-              </CardContent>
-            </Card>
-          </Paper>
-        </Grid>
-      </Grid>
-    )
-  }
-}
+			<Typography variant="subtitle1" align="right">
+				Last updated <b>6 hours ago</b>
```

Comment:
With the light theme variant this does not display

Reply:
Hmm, it displays for me:
![image](https://user-images.githubusercontent.com/7410405/80650458-a5580700-8a39-11ea-834b-8e6b80f84204.png)

As it's just a mockup of a feature that's not implemented on the backend yet, do you want to just remove it for now (and make a new issue if you want it added at some point)?

Reply:
Weird, I could have sworn I didn't see it. Yeah, for now, let's disable this and then actually get it implemented in the backend.